src/AppInstallerCLICore/Workflows/PortableFlow.cpp

```diff
@@ -452,10 +452,28 @@ namespace AppInstaller::CLI::Workflow
             if (symlinkPath.has_value())
             {
                 const std::filesystem::path& symlinkPathValue = symlinkPath.value().GetValue<Value::Type::UTF16String>();
-
-                if (!std::filesystem::remove(symlinkPathValue))
+                if (std::filesystem::is_symlink(std::filesystem::symlink_status(symlinkPathValue)))
                 {
-                    AICLI_LOG(CLI, Info, << "Portable symlink not found; Unable to delete portable symlink: " << symlinkPathValue);
+                    const auto& targetPath = uninstallEntry[PortableValueName::PortableTargetFullPath];
+                    if (targetPath.has_value())
+                    {
+                        const std::filesystem::path& symlinkTargetPath = std::filesystem::read_symlink(symlinkPathValue);
+                        const std::filesystem::path& targetPathValue = targetPath.value().GetValue<Value::Type::UTF16String>();

```

Comment:
```suggestion
                        const std::filesystem::path& targetPathValue = targetPath->GetValue<Value::Type::UTF16String>();
```
nit, but I don't know why you wouldn't want it to be easier to read...